experiments/configs/data_configs/hf_data.yml

```diff
@@ -0,0 +1,6 @@
+model_name: 'EleutherAI/pythia-1b'
+context_length: 2048
+dataset_name: 'suolyer/pile_pubmed-abstracts'
+dataset_args: {'split': 'test'}
```

Comment:
I've never tried this before, does it work? That's awesome

Reply:
Tested with more datasets and it works fine 👍